pkg/apis/core/annotation_key_constants.go

```diff
@@ -101,4 +101,7 @@ const (
 	// This annotation will be used to compute the in-cluster network programming latency SLI, see
 	// https://github.com/kubernetes/community/blob/master/sig-scalability/slos/network_programming_latency.md
 	EndpointsLastChangeTriggerTime = "endpoints.kubernetes.io/last-change-trigger-time"
+
+	// TODO(dyzz) Comment
+	MigratedPluginsAnnotationKey = "storage.alpha.kubernetes.io/migrated-plugins"
```

Comment:
seems reasonable for a new core API annotation to receive it's comment as part of the PR that adds the annotation.

Reply:
oversight, fixed